avocado/core/nrunner.py

```diff
@@ -193,7 +193,7 @@ def is_kind_supported_by_runner_command(self, runner_command):
                                        stdin=subprocess.DEVNULL,
                                        stdout=subprocess.PIPE,
                                        stderr=subprocess.DEVNULL)
-        except FileNotFoundError:
+        except (FileNotFoundError, PermissionError):
```

Comment:
@clebergnu , those are the detailed information that I was talking about during the last comment. Return `False` here, instead of raising exceptions will hide those information. I think that is important to proper handle the errors.

But, ok, no problem. We can do this on a next round.

Reply:
Agreed. I'm adding to my next round here.